### DISCUSSION TITLE
Image preview for worker login

### DISCUSSION THREAD

**Discussion by keshavadk** [cvat-ai/cvat#4384] [Q&A]
After setting up the latest CVAT in my local machine, 
I logged in as admin and created a user under worker group. 
Created a task and under the task ,job/jobs are visible.

As an admin,If i assign a job to the worker in Assignee tab(in job level) as shown in the image below and if i login as worker , the image preview will not be shown, but able to open and annotate as worker and able to save.

![admin](https://user-images.githubusercontent.com/16021947/155528683-665aec99-feec-4ddf-8a83-b4437a5fde39.png)

![worker](https://user-images.githubusercontent.com/16021947/155528711-7817bda7-4f18-4479-a46d-f0e880329cb7.png)

But, If i assign a task to worker in task level , the image preview will be shown.

Can we know the reason?

Thank you.

**Comment by bsekachev** [comment]
Hi, @keshavadk 

It was fixed there #4319.

**Reply by keshva23** [reply]
Thank you for the reply @bsekachev .
